### PR TITLE
Improve login screen design

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
We've improved the design of the login screen to match with the general design used in Web.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7552

## Screenshots

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

### CODE REVIEW

**Review by kulmann — COMMENTED** [review]
Could you check if it's easy to do to also use the `Inter` font here?

File: services/idp/src/app.css [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+@font-face {
+  font-family: Inter;
+  src: url(./fonts/inter.ttf) format('truetype');
+  font-weight: 100 900;
+  font-style: oblique 0deg 12deg;
+}
+
 body {
-  font-family: 'Open Sans', sans-serif;
+  font-family: Inter, 'Open Sans', sans-serif;
```
Comment (kulmann) [inline services/idp/src/app.css:10]: Please also add
```css
font-feature-settings: "cv11";
```
see https://github.com/owncloud/owncloud-design-system/pull/2299

**Review by kulmann — APPROVED** [review]
👍

**Comment by tbsbdr** [comment]
Its ok as is - thanks!

generally: we should avoid placeholder. duplication is redundant, having examples like "john@doe.com" is for some ppl confusing. good read: https://www.nngroup.com/articles/form-design-placeholders/

**Pull request merged**